pkg/controller/command/verifiable/models.go

```diff
@@ -55,8 +53,8 @@ type ProofOptions struct {
 	SignatureType string `json:"signatureType,omitempty"`
 	// PrivateKey is used to sign instead of DID
 	PrivateKey string `json:"privateKey,omitempty"`
-	// DIDKeyID id of the public key associated with the private key
-	DIDKeyID string `json:"didKeyID,omitempty"`
+	// proofPurpose is purpose of the proof.
+	proofPurpose string
```

Comment:
this doesn’t need json annotation or exporting?

Reply:
No, it is always "authentication", so didn't want to confuse command users with this option